book/07-git-tools/sections/revision-selection.asc

```diff
@@ -276,27 +276,27 @@ Date:   Fri Nov 7 13:47:59 2008 -0500
+Maaari mo ring pagsamahin ang mga syntax na ito -- maaari kang makakuha ng pangalawang magulang sa nakaraang reperensiya (ipagpapalagay na ito ay na-merge commit) sa pamamagitan ng paggamit ng `HEAD~3^2`, at iba pa.
 
 [[_commit_ranges]]
-==== Commit Ranges
+==== Mga Lawak ng Commit
 
-Now that you can specify individual commits, let’s see how to specify ranges of commits.
-This is particularly useful for managing your branches -- if you have a lot of branches, you can use range specifications to answer questions such as, ``What work is on this branch that I haven’t yet merged into my main branch?''
+Ngayon na magagawa mo na ang pagtukoy sa indibidwal na mga commit, tingnan nati kung paano magtukoy ng mga lawak sa mga commit.
+Ito ay partikular na kapaki-pakinabang para sa pagmamahala ng iyong mga branch -- kung mayroon kang maraming mga branch, maaari mong gamitin ang lawak ng pagtutukoy upang sagutin ang mga tanong tulad ng, ``Anong trabaho ang nasa branch na ito na hindi ko pa nai-merge sa aking pangunahing branch?''
 
-===== Double Dot
+===== Dobleng Dot
 
-The most common range specification is the double-dot syntax.
-This basically asks Git to resolve a range of commits that are reachable from one commit but aren’t reachable from another.
-For example, say you have a commit history that looks like <<double_dot>>.
+Ang pinaka-karaniwang na lawak na pagtutukoy ay ang doubleng-dot na syntax.
+Ito talaga ay nagtatanong sa Git na malutas ang lawak ng mga commit na mapupuntahan mula sa isang commit ngunit hindi maabot mula sa kabila.
+Halimbawa, sabihin na mayroon kang isang kasaysayan ng Git na mukhang ganito <<double_dot>>.
 
 [[double_dot]]
-.Example history for range selection.
-image::images/double-dot.png[Example history for range selection.]
+.Halimbawa ng kasaysayan sa lawak ng pagpili.
+image::images/double-dot.png[Halimbawa sa kasaysayan para sa lawak ng pagpipilian.]
 
-Say you want to see what is in your `experiment` branch that hasn’t yet been merged into your `master` branch.
-You can ask Git to show you a log of just those commits with `master..experiment` -- that means ``all commits reachable from experiment that aren’t reachable from master.''
-For the sake of brevity and clarity in these examples, the letters of the commit objects from the diagram are used in place of the actual log output in the order that they would display:
+Sabihin na gusto mong makita ang kung ano ang nasa iyong `eksperimento` na branch na hindi pa pinagsama sa iyong `master` na branch.
+Maaari kang magtanong sa Git upang ipakita sa iyo ang log ng mga commit na may `master..experiment` -- ibig sabihin ``ang lahat ng mga commit ay mapupuntahan mula sa eksperimento na hindi pa naabot mula sa master.''
+Para sa kapakanan ng kaiklian at kalinawan sa mga halimbawang ito, ang mga litra sa commit na mga bagay mula sa diagram ay ginagamit sa lugar sa aktwal na log output sa pagkakasunod-sunod na sila ay magpakita:
```

Comment:
- `litra` to `letra`
- you could change `diagram` to `dayagram`

Reply:
done change